### DISCUSSION TITLE
Option to hide afflictions from medical scanners and all viewing in XML

### DISCUSSION THREAD

**Discussion by Veritius** [FakeFishGames/Barotrauma#6219] [Ideas and suggestions]
Simply that. It would be extremely useful for more unusual uses of afflictions where it's better if it isn't shown.

**Comment by hubismal** [comment]
This already exists. Setting "showiconthreshold", "showicontoothersthreshold", and "showinhealthscannerthreshold" to a strength greater than the maxstrength ensures that nobody will ever see it.
You can see this utilized in the "disguised" affliction.

**Reply by Foxtrot39** [reply]
Is it possible for this to be class specific/skill based?

So the affliction may only be seen if your med skill is high enough?
